pom.xml

```diff
@@ -389,6 +389,9 @@
                 <groupId>org.eluder.coveralls</groupId>
                 <artifactId>coveralls-maven-plugin</artifactId>
                 <version>4.3.0</version>
+                <configuration>
+                    <sourceDirectories>circe-checksum/target/nar/nar-generated</sourceDirectories>
```

Comment:
any idea why we need to cover `nar-generated` here?

Reply:
Because coveralls.io plugin can't find the sources of generated classes.
I am wondering if would it be better to exclude all generated sources from code coverage.
I forgot to mark this PR as WIP

Reply:
Understand what sourcediectories mean. My question is why do we need to test coverage for generated code. I don’t see a reason to cover generated code, unless I missed anything.

Reply:
Sorry, I have misunderstood your comment.
I am working on skipping all generated code, it makes no sense to check it